evm/src/libraries/TransceiverStructs.sol

```diff
@@ -26,13 +26,13 @@ library TransceiverStructs {
 
     /// @dev Message emitted and received by the nttManager contract.
     ///      The wire format is as follows:
-    ///      - sequence - 8 bytes
+    ///      - id - 32 bytes
     ///      - sender - 32 bytes
     ///      - payloadLength - 2 bytes
     ///      - payload - `payloadLength` bytes
     struct NttManagerMessage {
         /// @notice unique sequence number
```

Comment:
do we still call this a unique _sequence_ number? since on solana side it's not incremental

Reply:
good point, let me fix